app/helpers/application_helper.rb

```diff
@@ -0,0 +1,11 @@
+module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = I18n.t "base_title"
+    if page_title.empty?
```

Comment:
1. Chị không khởi tạo giá trị cho page_title được ko em?

2. Tìm hiểu sự khác nhau của .blank?, .empty? và .nil? nhé.
Sau đó cân nhắc giúp chị em sử dụng .empty? ở line 5 như vậy là hợp lý chưa nhé

Reply:
Dạ em xin trả lời :
1. Không khởi tạo vẫn được vì khi chạy hàm sẽ tự tạo
2. Nên dùng .blank? , vì .blank? luôn trả về true khi chuỗi vào toàn là khoảng trắng , .empy? ko xử lí được . Nhờ vậy ta sẽ tránh đi các trường hợp bị dư các khoảng trắng ko cần thiết.

Reply:
Rứa chỉnh sửa đi nào